doc/modchannels.adoc

```diff
@@ -0,0 +1,82 @@
+= Modchannels
+
+allows server side mods(SSMs) comunication to client side mods(CSMs) and vice versa.
+
+== Server Side API
+
+=== Functions
+
+==== minetest.mod_channel_join(channel_name)
+
+* `channel_name`: string, modchannel name
+* returns ref for use with <<Methods>>
+* creates channel if not existing and joins the server on that channel
+* listen for messages on this channel with `minetest.register_on_modchannel_message`
+
+==== minetest.register_on_modchannel_message(function(channel_name, sender, message))
+
+* `channel_name`: string, modchannel name (already joined)
+* `sender`: string, empty if from a ssm, player name if from a client
+* `message`: string, message
+
+=== Methods
+
+==== leave()
```

Comment:
I think that showing the intended call is useful here.

```suggestion
NOTE: `obj` here means the object returned by `minetest.mod_channel_join`.

==== obj:leave()
```

Reply:
https://github.com/minetest/minetest_docs/pull/15/commits/fe0524f994a3c99b6f18e8bda4481e7592ee5419